friendshipper/src-tauri/src/main.rs

```diff
@@ -243,6 +264,14 @@ fn main() -> Result<(), CoreError> {
                     }
                 });
 
+                let (oidc_tx, oidc_rx) = std::sync::mpsc::channel::<OIDCTokens>();
+                let auth_handle = handle.clone();
+                thread::spawn(move || {
+                    while let Ok(tokens) = oidc_rx.recv() {
+                        auth_handle.emit_all("oidc-tokens", &tokens).unwrap();
```

Comment:
Is this `unwrap` safe? Are there any non-catastrophic conditions where the emit_all can fail but the tauri app doesn't exit completely?

Reply:
I'll handle it instead. I've never run into an issue here but famous last words